src/main/resources/runners/sigtest/pom.xml

```diff
@@ -127,22 +128,38 @@
                 </executions>
             </plugin>
             <plugin>
-                <groupId>org.netbeans.tools</groupId>
+                <!--                <groupId>org.netbeans.tools</groupId>-->
+                <groupId>jakarta.tck</groupId>
                 <artifactId>sigtest-maven-plugin</artifactId>
+                <version>2.2</version>
                 <executions>
-                  <execution>
-                    <id>default-cli</id>
-                    <phase>integration-test</phase>
-                    <goals>
-                      <goal>check</goal>
-                    </goals>
-                  </execution>
+                    <execution>
+                        <id>default-cli</id>
+                        <phase>integration-test</phase>
+                        <goals>
+                            <goal>check</goal>
+                            <!--                            <goal>generate</goal>-->
+                        </goals>
+                    </execution>
                 </executions>
                 <configuration>
-                  <action>strictcheck</action>
-                  <failOnError>true</failOnError>
-                  <packages>jakarta.batch.**</packages>
-                  <classes>${project.build.directory}/dependency</classes>
+                    <action>strictcheck</action>
+                    <failOnError>true</failOnError>
+                    <packages>jakarta.batch.**</packages>
+                    <classes>${project.build.directory}/dependency</classes>
+                    <!--                </configuration>-->
+                    <!--                <configuration>-->
+                    <!--                    <sigfile>${project.build.directory}/jakarta.batch.sig_${project.parent.version}</sigfile>-->
+                    <!--                    <sigfile>-->
+                    <!--                        ${project.build.directory}/sigtest-copy/sigtest/sigtest-1.6-batch.standalone.tck.sig-2.1-se11-OpenJDK-J9-->
+                    <!--                    </sigfile>-->
+                    <sigfile>
+                        ${project.build.directory}/sigtest-copy/sigtest/jakarta.sigtest-2.2-batch.standalone.tck.sig-2.1-se21-Temurin
```

Comment:
That path doesn't look right to me.

Reply:
I guess this is related: https://github.com/jakartaee/batch-tck/pull/72#discussion_r1541492266